### PR TITLE
fix(graphql): Allow including 'File' scalar by default to be disabled

### PR DESCRIPTION
**Pull request by Josh-Walker-GM**
There was a problem introduced in v8 when we included the `File` scalar by default. This meant a custom implementation by the user could be clobbered by the new default. This change allows the user to supply config to disable including it by default.

This is not how I would have loved to have done things here. Config in two places is rubbish but given the organisation of this currently it was generally unavoidable.

### CODE REVIEW

File: docs/docs/graphql.md [hunk 1]
```diff
@@ -689,6 +689,43 @@ api | - deletePost Mutation
...
+     loggerConfig: { logger, options: {} },
+     directives,
+     sdls,
+     services,
+     onException: () => {
+       // Disconnect from your database with an unhandled exception.
+       db.$disconnect()
+     },
+     // highlight-start
+     includeScalars: {
```
Comment (dthyresson) [inline docs/docs/graphql.md:720]: This is better than what I had - the config is at least consistent looking between toml and Yoga config. Nice.

**Review by dthyresson — COMMENTED** [review]
Thanks for this! @Josh-Walker-GM 

LGTM and will now let people who have had a Prisma File model opt out effectively of Uploads.

My only question is if in The Default Scalars docs or in Storage we should mention that if you opt out of the file scalar, your project will not be able to use Redwood Storage or any GraphQL file upload.

**Review by Tobbe — DISMISSED** [review]
Can we please discuss this one more round before merging?

**Comment by cannikin** [comment]
Love that this is still enabled by default and it's opt *out*!

**Comment by Josh-Walker-GM** [comment]
We decided we wanted to go ahead with this as a fix for the reported problem.

**Pull request merged**